locales/en/app.ftl

```diff
@@ -459,3 +459,305 @@ get-your-report-and-sign-up= Get your report and sign up for alerts.
+
+link-change-primary = Change primary email address
+
+remove-fxm = Remove {-product-name}
+remove-fxm-blurb = Turn off {-product-name} alerts. Your {-brand-fxa} will remain active, and you may receive 
+  other account-related communications.
+
+
+manage-email-addresses = Manage Email Addresses
+
+latest-breach-link = See if you were in this breach
+
+welcome-back = Welcome back, { $userName }!
+welcome-user = Welcome, { $userName }!
+
+
+breach-alert-subject = {-product-name} found your email in a new data breach.
+
+your-info-was-discovered-headline = Your information was discovered in a new data breach.
+
+your-info-was-discovered-blurb = You’re signed up to receive {-product-name} alerts 
+  when your email appears in a data breach. Here’s what we know about this breach.
+
+what-to-do-after-breach = What to do after a data breach
+
+ba-next-step-1 = Change your password to a strong, unique password.
+ba-next-step-blurb-1 =
+  A strong password uses a combination of upper and lowercase letters, 
+  special characters, and numbers. It doesn’t contain personal info like 
+  your address, birthday, or family names.
+
+ba-next-step-2 = Stop using that exposed password entirely.
+ba-next-step-blurb-2 =
+  Cyber criminals could find your password on the dark web and use it 
+  to log in to your other accounts. The best way to protect your accounts 
+  is to use unique passwords for each one.
+
+
+ba-next-step-3 = Get help creating better passwords and keeping them safe.
+ba-next-step-blurb-3 =
+  Use a password manager to create strong, unique passwords. Password managers securely store all your 
+  logins so you can access them across all your devices.
+
+faq1 = I don’t recognize this company or website. Why am I in this breach?
+faq2 = Why did it take so long to notify me of this breach?
+faq3 = How do I know this is a legitimate email from {-product-name}?
+
+
+new-breaches-found =
+  { $breachCount ->
+     *[one] { $breachCount } NEW BREACH FOUND
+      [other] { $breachCount } NEW BREACHES FOUND
+  }
+
+sign-up-headline-1 = Get ongoing alerts with a {-brand-fxa}.
+sign-up-for-alerts = Sign up for Alerts
+account-not-required = {-brand-name} browser not required for a {-brand-fxa}. You may receive info about {-brand-Mozilla} services.
+
+
+get-alerted = Get alerted about new breaches.
+
+was-your-info-exposed = Was your info exposed in the { $breachName } data breach?
+find-out-if = Find out if your data was exposed in this breach.
+
+fb-not-comp = This email did not appear in the { $breachName } breach.
+fb-comp = This email appeared in { $breachCount } known data breaches, including { $breachName }. 
+fb-comp-only = This email appeared in the { $breachCount } breach.
+
+other-breaches-found = However, it did appear in 
+  { $breachCount ->
+    [one] { $breachCount } other breach.
+   *[two] { $breachCount } other breaches. 
+  }
+
+fb-comp-and-others = This email appeared in { $breachCount } known data breaches, including { $breachName }.
```

Comment:
This needs a plural.

Reply:
This still need to be address (we need a proper plural form, with ONE and OTHER for English).